driver/pace/driver/driver.py

```diff
@@ -108,6 +108,10 @@ def total_time(self) -> timedelta:
             days=self.days, hours=self.hours, minutes=self.minutes, seconds=self.seconds
         )
 
+    def n_timesteps(self) -> int:
+        """Computing how many timestep required to carry the simulation."""
+        return int(self.total_time.seconds / self.timestep.seconds)
```

Comment:
Good question. My instinct is "if you don't have enough time or timestep required is too large, don't do anything" but I could be wrong _and_ I could emit a warning at the very least.

I'll also use round instead of a cast. Sorry bad C dev reflexes.

What make sense to you, floor? Ceil? Round?

Reply:
Great, round is perfect. And great idea on adding a warning!